MainWindow.xaml

```diff
@@ -5,56 +5,92 @@
-        <TextBlock x:Name="Game" Grid.Column="2" Grid.ColumnSpan="4" Grid.Row="18" FontSize="18" TextAlignment="Center" VerticalAlignment="Center"></TextBlock>
-        <Button x:Name="Install" Grid.Column="6" Grid.Row="18" FontSize="18" Click="InstallMod">Install</Button>
+                    <Label Foreground="{StaticResource ForegroundBlueStandardBrush}" FontFamily="./Fonts/#Poppins" FontWeight="DemiBold" FontSize="12" VerticalAlignment="Bottom" HorizontalAlignment="Left" >Author(s)</Label>
+                    <TextBox x:Name="Authors" FontFamily="./Fonts/#Poppins" Background="{StaticResource BackgroundLightBrush}" FontSize="14" Padding="4" IsEnabled="False" Margin="12,0,22,0" Foreground="{StaticResource ForegroundWhiteStandardBrush}"></TextBox>
+                </StackPanel>
+                <Image x:Name="Cover" Grid.Column="1" Height="120" Width="120" HorizontalAlignment="Right" Source="Images/placeholder.png" Margin="0,0,12,0"></Image>
+            </Grid>
+
+            <Label Foreground="{StaticResource ForegroundBlueStandardBrush}" FontFamily="./Fonts/#Poppins" FontWeight="DemiBold" FontSize="12" VerticalAlignment="Bottom" HorizontalAlignment="Left">Description</Label>
+            <TextBox x:Name="Description" FontFamily="./Fonts/#Poppins" MinLines="3" Background="{StaticResource BackgroundLightBrush}" FontSize="14" Padding="4" TextWrapping="Wrap" Margin="12,0,12,0" IsEnabled="False" Foreground="{StaticResource ForegroundWhiteStandardBrush}"></TextBox>
+
+            <Label Foreground="{StaticResource ForegroundBlueStandardBrush}" FontFamily="./Fonts/#Poppins" FontWeight="DemiBold" FontSize="12" VerticalAlignment="Bottom" HorizontalAlignment="Left">Mod directory</Label>
+            <TextBox x:Name="ModDir" FontFamily="./Fonts/#Poppins" Background="{StaticResource BackgroundLightBrush}" FontSize="14" Padding="4" IsEnabled="False" Margin="12,0,12,0" Foreground="{StaticResource ForegroundWhiteStandardBrush}"></TextBox>
+
+            <Label Foreground="{StaticResource ForegroundBlueStandardBrush}" FontFamily="./Fonts/#Poppins" FontWeight="DemiBold" FontSize="12" VerticalAlignment="Bottom" HorizontalAlignment="Left">Install directory</Label>
+            <StackPanel Orientation="Horizontal">
+                <TextBox HorizontalAlignment="Left" FontFamily="./Fonts/#Poppins" Background="{StaticResource BackgroundLightBrush}" Width="260" x:Name="InstallDir" Grid.Column="1" Grid.ColumnSpan="5" Grid.Row="16" FontSize="14" Padding="4" Margin="12,0,12,0" Foreground="{StaticResource ForegroundWhiteStandardBrush}"/>
+                <Button x:Name="Browse" Background="{StaticResource BackgroundLightBrush}" FontSize="12" Foreground="{StaticResource ForegroundWhiteImportantBrush}" MinWidth="40" FontFamily="./Fonts/#Poppins" Click="BrowseFolder">...</Button>
+            </StackPanel>
+
+
+            <TextBlock x:Name="Game" FontFamily="./Fonts/#Poppins" FontSize="1" TextAlignment="Center" VerticalAlignment="Center"></TextBlock>
```

Comment:
This crunches the Game field and makes it impossible to display under normal behavior, and should be fixed by adjusting the layout of the window.